### DISCUSSION TITLE
Suggestions for a few additional fields and event types

### DISCUSSION THREAD

**Discussion by fludstank** [MDeLuise/plant-it#97] [Feature Request]
### Avoid duplicated feature requests

- [X] There are no open or closed feature requests that are related to this request

### Description

First of all, this has been absolutely fantastic and really helpful for keeping track of my plants. A couple of elements that would be great in the Plant Stats field that I've been using the notes for, but might be helpful to have as their own fields have been

- purchase price
- where purchased/seller info/person who gifted
- physical location (which room, tray number, home, office, offsite location, etc)
- link the plant/tag as a propagation of another in the collection

Related to the last one, it would also be helpful to have an event for propagating. Unrelated to that would be events for pruning and repotting (I've been using transplanting but idk if that's technically the same or not).

Even if these enhancements don't make it, I'm still going to be a huge fan and will be recommending it to my self hosting plant enthusiast friends!

**Comment by MDeLuise** [comment]
Hi @fludstank,

Thank you so much for your kind words and for being a part of the project! I'm thrilled to hear that it has proven helpful in keeping track of your plants.
Your feature requests are truly appreciated, and I'm happy to work on enhancing the platform based on your input.

To address the "physical location" and "seller info" aspects, I'm considering the addition of simple text fields. This way, you'll have the flexibility to input the details that matter most to you. If you have any specific ideas about the implementation, please feel free to share them with me.

Moreover, the other suggestions you provided, such as linking a plant as a propagation, purchase price, and introducing dedicated events for propagating, pruning, and repotting, are fantastic ideas that will certainly add more granularity to the tracking of the plants' lifecycle.

**Reply by fludstank** [reply]
Outstanding! That sounds really great - thank you for such kind words - it's just going to get better and better!

**Comment by MDeLuise** [comment]
Hi @fludstank,

The requested feature has been implemented in PR #115 and will be included in the upcoming release of the project.
Initially, propagation functions as a simple event like others.
However, future PRs, will introduce a "propagating relationship." (feature tracked in issue #114). This enhancement will enable users to navigate from one plant to another that is propagated from the first, thereby keeping track of the "sons" of a given plant.

Thank you for your contribution to the project! 🌱

Here are a few screenshot of the next project release
<p align="center">
  <img src="https://github.com/MDeLuise/plant-it/assets/66636702/d7f5cfbb-3800-44f6-9379-fc70d7b040f7" width="33%" />
  <img src="https://github.com/MDeLuise/plant-it/assets/66636702/479e21a5-d9a9-4ea8-9cb0-9b4847b26e77" width="33%" />
  <img src="https://github.com/MDeLuise/plant-it/assets/66636702/8e51c7bf-1301-4a88-b6ad-08069f14d44b" width="33%">
</p>
